deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -176,3 +272,7 @@ export SUMOLOGIC_ACCESSKEY=
 export SUMOLOGIC_ACCESSID=
 
 bash /etc/terraform/custom.sh
+
+{{- if .Values.sumologic.setup.dashboards.enabled }}
+can_create_pvc
+{{- end }}
```

Comment:
Maybe because of waiting, this should be run as a separate process in the background.